app/src/main/java/com/infomaniak/mail/utils/MessageBodyUtils.kt

```diff
@@ -70,6 +72,12 @@ object MessageBodyUtils {
             block = {
                 // Do not nest jsoupParseWithLog and measureAndLogMemoryUsage so logs are independent from one another
                 val htmlDocument = jsoupParseWithLog(bodyContent)
+
+                if (htmlDocument.text().length > MESSAGE_LENGTH_LIMIT) {
+                    val content = bodyContent.substring(0, MESSAGE_LENGTH_LIMIT)
```

Comment:
Do we really need the substring? If we skip the processing of the mail, no need to substring it anymore right? The problem has already been avoided

Reply:
It is needed to prevent an Out Of Memory error, resulting in a crash